test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -202,54 +203,54 @@ void ProtoConverter::visit(BinaryOp const& _x)
 
 void ProtoConverter::visit(VarDecl const& _x)
 {
-	m_output << "let x_" << m_numLiveVars << " := ";
+	string varName = "x_" + to_string(counter());
+	m_output << "let " << varName << " := ";
```

Comment:
Would be nice to have little helper functions for this in the future ;)

Reply:
I can wrap the first line in a utility function. However, I need `varName` for later though.